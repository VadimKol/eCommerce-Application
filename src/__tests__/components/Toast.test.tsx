import { render } from '@testing-library/react';

import { Toast } from '@/components/toast/Toast';

it('renders correctly', () => {
  const { container } = render(<Toast />);
  expect(container).toMatchSnapshot();
});
