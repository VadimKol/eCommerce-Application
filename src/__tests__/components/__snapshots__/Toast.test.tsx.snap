// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="Toast"
  >
    <div
      class="Toastify"
    />
  </div>
</div>
`;
